samples/ambient-argo/application/ingress-gateway.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: bookinfo-gateway
+spec:
+  gatewayClassName: istio
+  listeners:
+  - name: http
+    port: 80
+    protocol: HTTP
+    allowedRoutes:
+      namespaces:
+        from: Same
+---
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: bookinfo
+spec:
+  parentRefs:
+  - name: bookinfo-gateway
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /locust
+    filters:
+    - type: URLRewrite
+      urlRewrite:
+        path:
+          type: ReplacePrefixMatch
+          replacePrefixMatch: /
+    backendRefs:
+    - name: loadgen
+      port: 8089
+  - matches:
+    - path:
+        type: Exact
+        value: /stats/reset
+    backendRefs:
+    - name: blackhole
+      port: 1234
```

Comment:
IIUC this may be more appropriate as an AuthorizationPolicy?

Reply:
That was my first thought too.  Except loadgen doesn't have a waypoint.  I can just drop part from the repo.  It's useful for my demo (attendees can view the load generator in real time), but it's really not useful as a part of the reference arch.

Reply:
Isn't the point of this to block access from ingress? You want access from in-cluster. So you could block it at ingress.

Not a big deal

Reply:
Removed loadgen.